www/base/src/app/common/directives/buildsummary/buildsummary.directive.coffee

```diff
@@ -71,6 +72,9 @@ class _buildsummary extends Controller('common')
         @isSummaryLog = (log) -> 
```

Comment:
So this function is not necessary anymore?

Reply:
https://github.com/buildbot/buildbot/blob/27abf3e0c95ddd64f67be3c87d90a323c1560977/www/base/src/app/common/directives/buildsummary/buildsummary.directive.coffee#L74-L78
Still used here, to pop summary log to the top of the step view.